neuralpp/symbolic/eliminator.py

```diff
@@ -0,0 +1,85 @@
+from .expression import QuantifierExpression, Context, Expression, Constant, AbelianOperation, Variable, \
+    FunctionApplication
+from .basic_expression import BasicQuantifierExpression
+from .sympy_expression import SymPyExpression
+from .interval import ClosedInterval, DottedIntervals, from_constraint
+from .constants import if_then_else
+import neuralpp.symbolic.functions as functions
+from functools import reduce
+from typing import Optional, Callable
+import operator
+
+
+def _eliminate_interval(operation: AbelianOperation, index: Variable, interval: ClosedInterval, body: Expression) \
+        -> Expression:
+    # try to solve symbolically
+    result = _symbolically_eliminate(operation, index, interval, body)
+    if result is not None:
+        return result
+
+    # TODO: enable this
```

Comment:
temporarily comment out this because for our current scope, symbolic eliminate should always work